hydra-node/test/Hydra/Network/AuthenticateSpec.hs

```diff
@@ -33,6 +33,8 @@ spec = parallel $ do
           receivedMessages <- newTVarIO []
 
           withAuthentication
+            @(Message SimpleTx)
+            @(Message SimpleTx)
```

Comment:
Why do we suddenly need these type assertions?

Reply:
• Ambiguous type variable ‘outbound2’ arising from a use of ‘withAuthentication’
      prevents the constraint ‘(SignableRepresentation
                                  outbound2)’ from being solved.